tests/php/Extension/MemberExtensionTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace SilverStripe\MFA\Tests\Extension;
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\Group;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+
+/**
+ * Class MemberExtensionTest
+ *
+ * @group wip
```

Comment:
Is it still wip? This docblock doesn't provide any value actually, maybe just delete it